src/sections/referrals/components/CodePreview/CodePreview.tsx

```diff
@@ -20,24 +20,35 @@ type Props = {
   disabled?: boolean
 }
 
+const IS_PROD = import.meta.env.PROD
```

Comment:
is that a valid variable?

Reply:
there is a VITE_ENV=production variable. It's better I think

Reply:
true, didnt see it, fixed